cms/djangoapps/contentstore/api/views/course_quality.py

```diff
@@ -82,14 +92,17 @@ def get(self, request, course_key):
         """
         Returns validation information for the given course.
         """
-        def _execute_method_and_log_time(func, *args):
+        def _execute_method_and_log_time(log_time, func, *args):
             """
             Call func passed in method with logging the time it took to complete.
             Logging is temporary, we will remove this once we get required information.
             """
-            start_time = time.time()
-            output = func(*args)
-            log.info('[%s] completed in [%f]', func.__name__, (time.time() - start_time))
+            if log_time:
```

Comment:
lets change it to `course_key_harvard ` or  `course_key_harvard ` to `log_time ` to improve readability.